src/EditorFeatures/CSharpTest/Diagnostics/RemoveUnnecessaryCast/RemoveUnnecessaryCastTests.cs

```diff
@@ -59,7 +59,7 @@ static void Main()
     {
         int x = 2;
         int i = 1;
-        Goo((x < i), x > (2 + 3));
+        Goo(x < (i), x > (2 + 3));
```

Comment:
is () around "I" there due to ","?

Reply:
The parens are there because if we removed them hte above would parse as a generic invocation.  i.e. as ```A<X,Y>(...)```.  it's a pathological case which no user would really ever hit.  but we ensure that we don't causing parsing to change meaning.